### PR TITLE
N49: Graviton - The Great Sand Simulator

### PR DESCRIPTION
**Pull request by hakolao**
Part of #1440

### CODE REVIEW

File: content/news/049/index.md [hunk 1]
```diff
@@ -67,6 +67,37 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+### [Graviton - The Great Sand Simulator](http://s.team/a/2137280)
+
+![Graviton Sand Simulator Image](graviton.gif)
+
+[Graviton - The Great Sand Simulator](https://www.gravitongame.art/)
+([Discord](https://discord.gg/3MyPaDagsd),
+[Youtube](https://www.youtube.com/watch?v=X_O90KyEt8o)) by
+[@hakolao](https://github.com/hakolao/)
+is a falling sand simulator in which you can define your own interactive
+sand rules.
```
Comment (AngelOnFira) [inline content/news/049/index.md:79]: We move links to the bottom of the section for cleanliness 👍 if you could switch it over, that would be appreciated!
```
[This blog post][graviton-blog-post]

...

[graviton-blog-post]: https://example.com
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]

**Comment by hakolao** [comment]
Ok, done :)

File: content/news/049/index.md (hunk 1 above)
Comment (ozkriff) [reply content/news/049/index.md]: (@hakolao this was about using reference-style Markdown formatting for links, the placement of them was fine)

**Review by ozkriff — APPROVED** [review]
:+1: (i've made a couple of fmt/style fixes for consistency with the rest of the newsletter)

**Comment by ozkriff** [comment]
Thanks for the PR (and sorry for the review delays)!

**Comment by hakolao** [comment]
Thanks you!

**Pull request merged**